app/src/main/res/layout/dashboard_funding_view.xml

```diff
@@ -0,0 +1,123 @@
+    android:gravity="center_horizontal"
+    android:layout_marginBottom="@dimen/grid_new_3"
+    style="@style/BodyPrimary"
+    tools:text="pledged of $12,000"/>
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:paddingLeft="@dimen/grid_new_5"
+    android:paddingRight="@dimen/grid_new_5">
+
+    <LinearLayout
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:orientation="vertical"
+      android:paddingLeft="@dimen/grid_7"
+      android:paddingRight="@dimen/grid_7">
+
+      <TextView
+        android:id="@+id/creator_dashboard_percent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/BodyPrimary"
+        tools:text="100%"/>
+
+      <TextView
+        android:id="@+id/creator_dashboard_percent_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/discovery_baseball_card_stats_funded"
+        tools:text="funded"/>
+
+    </LinearLayout>
+
+    <LinearLayout
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:orientation="vertical">
+
+      <TextView
+        android:id="@+id/creator_dashboard_backer_count"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/BodyPrimary"
+        tools:text="100"/>
+
+      <TextView
+        android:id="@+id/creator_dashboard_backer_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/BodySecondary"
+        android:layout_marginEnd="@dimen/grid_new_4"
+        android:text="@string/discovery_baseball_card_stats_backers"
+        tools:text="backers"/>
+
+    </LinearLayout>
+
+    <LinearLayout
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:orientation="vertical">
+
+      <TextView
+        android:id="@+id/creator_dashboard_time_remaining"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        style="@style/BodyPrimary"
+        tools:text="3"/>
+
+      <TextView
+        android:id="@+id/creator_dashboard_time_remaining_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        tools:text="secs to go"/>
+
+    </LinearLayout>
+
+  </LinearLayout>
+
+
+  <Button
+    android:id="@+id/creator_view_project_button"
+    style="@style/BlueRoundedButton"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/grid_new_7"
+    android:layout_marginTop="@dimen/grid_new_2"
+    android:layout_marginRight="@dimen/grid_new_5"
+    android:layout_marginLeft="@dimen/grid_new_5"
+    android:text="@string/View"
+    tools:text="Go to project" />
+</LinearLayout>
```

Comment:
Wonder if there's some new line setting that we should apply for consistency? I believe we used to ensure there was a new line at the end of every file, maybe there's a linter rule for this we could add